mobile/src/screens/profile-settings/profile-settings.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import LinearGradient from 'react-native-linear-gradient';
```

Comment:
I see that we are using this component in lots of files, let's create a custom component for that and use it everywhere